bin/prettier.js

```diff
@@ -185,6 +187,7 @@ if (argv["help"] || (!filepatterns.length && !stdin)) {
       "  --print-width <int>      Specify the length of line that the printer will wrap on. Defaults to 80.\n" +
       "  --tab-width <int>        Specify the number of spaces per indentation-level. Defaults to 2.\n" +
       "  --use-tabs               Indent lines with tabs instead of spaces. Defaults to false.\n" +
+      "  --no-semi                Do not print semicolons, except at the beginning of lines which may need them. Defaults to false.\n" +
```

Comment:
This is inconsistent with `--bracket-spacing` and `--color`. I think it should be written as `--semi` here, not `--no-semi` (or the others should be changed as well).